Docs/Configuration.tex

```diff
@@ -6117,19 +6115,19 @@ \subsection{Audio Properties}\label{uefiaudioprops}
   \textbf{Failsafe}: \texttt{0}\\
   \textbf{Description}: Minimal heard volume level from \texttt{0} to \texttt{100}.
 
-  Screen reader will use this volume level, when the calculated volume level is
-  less than \texttt{MinimumVolume}. Boot chime sound will not play if the calculated
-  volume level is less than \texttt{MinimumVolume}.
+  Screen reader will use this volume level when the calculated volume level is lower
+  than \texttt{MinimumVolume}. The boot chime will not sound if the calculated
```

Comment:
will not be played? I am a bit surprised sound verb is used to describe audio playback.

Reply:
A chime sounds ... like a bell rings.

Reply:
Well, ok, but it is play everywhere below.

Reply:
Changed to be same as others